src/readii_2_roqc/utils/settings.py

```
@@ -0,0 +1,168 @@
+
+    # Load the requested image processing settings from configuration
+    _regions, _permutations, crop, resize = get_readii_settings(dataset_config)
+
+    try:
+        # Path to find existing readii index output for checking existing outputs
+        if crop != "" and resize != []:
+            readii_index_filepath = readii_image_dir.glob(f"{crop}_{get_resize_string(resize)}/readii_{dataset_name}_index.csv").__next__()
+        else:
+            readii_index_filepath = readii_image_dir.glob(f"original_*/readii_{dataset_name}_index.csv").__next__()
+    except StopIteration:
+        message = f"No READII index file was found for the specified settings"
```

Ruff (0.13.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)